src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let arrayString = '';
+    const separatorString = String(separator);
```

Comment:
separator always will be a string, so you can remove line 9, and on line 17 change separatorString to separator
But if it works, it works! good job🤝👍

Reply:
It's good comment, remove it 

[5-ways-to-convert-a-value-to-string-in-javascript](https://medium.com/dailyjs/5-ways-to-convert-a-value-to-string-in-javascript-6b334b2fc778)